### PR TITLE
added featured template ui

### PR DESCRIPTION
**Pull request by MeenuKumari28**
- added featured template page
- added linktree templates
- added coming soon templates

### CODE REVIEW

File: records/templates-list.ftd [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+-- record template-details:
+optional ftd.image-src screenshot:
+optional string template-name:
+optional body desc:
+optional string updated-on:
+optional string preview-url:
+optional string github-url:
+optional string features:
+optional string pages:
```
Comment (amitu) [inline records/templates-list.ftd:22]: THese two should be `string list features` etc, not `optional string features`.

**Comment by amitu** [comment]
@MeenuKumari28 if we do this the size of package will go up as we add mode packages as we would be storing a lot of preview images. Also the target package will not be able to update their preview without updating this repo. 

Lets remove all preview images and put preview in each repo. Similarly lets add all the meta data about a package in the package itself, eg the name of the package, the author, the updated on etc.

We can create a record, say `design-system.package-info`:

```ftd
-- record package-info:
caption name:
ftd.image-src preview:
string updated-at:
```

Actually you have already defined a record, template-details, this is not a great name, so lets rename it to `package-info`.

**Review by yashveer29 — APPROVED** [review]

**Comment by amitu** [comment]
@MeenuKumari28 what is the status of this?

File: FASTN.ftd [hunk 2]
```diff
@@ -17,6 +17,18 @@ provided-via: ui.fifthtry.com/ds
...
+-- fastn.dependency: meenukumari28.github.io/linktree-template-3
+-- fastn.dependency: meenukumari28.github.io/linktree-template-4
+-- fastn.dependency: meenukumari28.github.io/coming-soon-1
+-- fastn.dependency: meenukumari28.github.io/coming-soon-2
+-- fastn.dependency: meenukumari28.github.io/coming-soon-3
+-- fastn.dependency: meenukumari28.github.io/coming-soon-4
+-- fastn.dependency: meenukumari28.github.io/coming-soon-5
+-- fastn.dependency: meenukumari28.github.io/coming-soon-6
+-- fastn.dependency: meenukumari28.github.io/coming-soon-7
+-- fastn.dependency: meenukumari28.github.io/coming-soon-8
```
Comment (amitu) [inline FASTN.ftd:31]: @MeenuKumari28 we have to move all these to fastn-community github org. Do you have permission to create a new repo in our org, else I will do it. Also need better names than coming-soon-1 etc. coming-soon-ross, coming-soon-phoebe etc you can do. And similarly charlie, alan, jake, rose for the linktree ones.

Comment (amitu) [reply FASTN.ftd:31]: Also these files should not exist in this package: `assets/templates/coming-soon-1.PNG` You should put them in say `-- import: meenukumari28.github.io/coming-soon-8/assets as coming-soon-8-assets`, and use it as `src: $coming-soon-8-assets.files.preview.png`.

**Pull request merged**